components/servo/lib.rs

```diff
@@ -333,6 +334,11 @@ where
                 clear_color[2] as f32,
                 clear_color[3] as f32,
             );
+            let upload_method = if webrender_gl.get_string(RENDERER).starts_with("ANGLE") {
```

Comment:
can't we create something like Firefox [is_glcontext_angle](https://searchfox.org/mozilla-central/source/gfx/layers/wr/WebRenderBridgeParent.cpp#80) ?

Reply:
Probably with changes to `surfman` otherwise through GL bindings I didn't find another way. `webrender` uses the same check when testing for ANGLE: https://github.com/servo/webrender/blob/c0bcdd024adac1297ceb2f34a2de46731243c970/webrender/src/device/gl.rs#L1724-L1725

Reply:
Yes I see, just that we may end up with lot of `starts_with("ANGLE")` checks in code